music/src/core/audio_utils.ts

```diff
@@ -453,7 +504,7 @@ function outerSubtract(arr: Float32Array, arr2: Float32Array): Float32Array[] {
 }
 
 function pow(arr: Float32Array, power: number) {
-  return arr.map(v => Math.pow(v, power));
+  return arr.map((v) => Math.pow(v, power));
```

Comment:
What code formatter are you using? Is it the project default one (clang-format)? there's a lot of these meaningless formatting changes that I'm not sure where they're coming from...

Reply:
I've re-saved all the files I touched with the clang-formatter and if you still see a lot of changes, it's because of the tonejs and tfjs upgrade. 

The change that touched the most file was the deprecation of `toMaster`, which required me to change it to `toDestination`.

Reply:
Thanks! Looks good now!